VocaDbModel/Database/Queries/AlbumQueries.cs

```diff
@@ -363,9 +365,10 @@ public AlbumDetailsContract GetAlbumDetails(int id, string hostname)
 
 				var stats = session.Query<Album>()

```

Comment:
You should move the comment loading and hit counting to separate calls.

The reason it's done like this is to load both of these stats using a single call, but because of the ToArray() you added, it's loading them separately anyway, so it's unnecessary. Best would be to cache these values also.